fe/fe-core/src/main/java/com/starrocks/qe/Coordinator.java

```diff
@@ -1127,7 +1127,7 @@ private void setGlobalRuntimeFilterParams(FragmentExecParams topParams, TNetwork
                     probeParamList.add(probeParam);
                 }
                 if (usePipeline && kv.getValue().isBroadcastJoin() && kv.getValue().isHasRemoteTargets()) {
-                    broadcastGRFProbersMap.put(kv.getKey(), probeParamList);
+                    broadcastGRFProbersMap.computeIfAbsent(kv.getKey(), k -> new ArrayList<>()).addAll(probeParamList);
```

Comment:
when does it happens? it will affect all remote broadcast join?

Reply:
some corner cases,  for examples, rf are pushed down two sides of an Operator.